content/source/docs/cloud/registry/publish.html.md

```diff
@@ -1,22 +1,22 @@
 ---
 layout: "cloud"
-page_title: "Publishing Private Modules - Private Module Registry - Terraform Cloud and Terraform Enterprise"
+page_title: "Publishing Private Modules - Private Registry - Terraform Cloud and Terraform Enterprise"
 description: |-
   Learn to prepare private modules for publishing, add them to the registry, and release new versions.
 ---
 
 [vcs]: ../vcs/index.html
 
-# Publishing Modules to the Terraform Cloud Private Module Registry
+# Publishing Modules to the Terraform Cloud Private Registry
 
 > **Hands-on:** Try the [Share Modules in the Private Module Registry](https://learn.hashicorp.com/tutorials/terraform/module-private-registry-share) tutorial on HashiCorp Learn.
```

Comment:
The title of this tutorial might change with the release

Reply:
oh okay good to know! I'll check with rita!